project_hr/views/project_task_views.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2018 Tecnativa - Pedro M. Baeza
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
-
+<!--
+    Copyright 2018 Tecnativa - Pedro M. Baeza
+    Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+-->
```

Comment:
Please do not reformat this code chunk. Just add a line with your copyright if you want.

This was following what's OCA standard, [as you can find in the official module template](https://github.com/OCA/maintainer-tools/blob/771508e99876115c418888bbac7899174662482b/template/module/views/report_name.xml#L2-L3).